portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,70 @@
+/* LINKS */
+
+a {
+  color: rgb(0, 180, 0);
+  text-decoration: none;
+ }
+
+a:hover {
+  color: rgb(0, 230, 0);
+  text-decoration: none;
+ }
+
+/* GENERAL */
+
+body {
+  background-color: black;
+  color: gainsboro;
+  font-family:  'IBM Plex Mono', 'Fira code', 'Courier New', Courier, monospace;;
+ }
+
+.container {
+  display: flex;
+  align-content: center;
+ }
+
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width:800px;
+ }
+
+.emphasis {
+  color: rgb(0, 170, 0)
 }
 
-#greeting-container {
-  margin-top: 20px;
+/* NAV BAR */
+
+nav {
+  padding-top: 3vh;
+  padding-bottom: 3vh;
+ }
+
+/* PROJECT SECTION */
+
+.project-wrapper {
+  visibility: hidden;
```

Comment:
Let's alphabetize the CSS declarations https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order